index.js

```diff
@@ -1642,7 +1642,7 @@ function unpackneg(r, p) {
   return 0;
 }
 
-function crypto_sign_open(msg, sm, pk) {
+function crypto_sign_open(sm, msg, pk) {
```

Comment:
msg, signedMessage, pk is actually correct (see sodium-native). output is always the first argument. could you change this back so we can get the other fixes in there :)